src/frontend/src/pages/PPQPage/PPQPage.tsx

```diff
@@ -11,20 +14,20 @@ function PPQLandingPage() {
       <div className="ppq-section results-wrapper">
         <div className="row what-is-a-ppq">
           <div className="col-md-6">
-            <h2>PIA Pathway Questionnaire</h2>
+            <h2>PIA Intake</h2>
```

Comment:
This heading needs a `font-weight: 700`

Reply:
i change it to h1, is that ok?

Reply:
Can we have a broader discussion on this. From my understanding we should set the style for H1, H2, p etc and never have to touch font styling ever again. Can we spend some time on defining this at the root level?
https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#good_semantics

Below are detailed links on how we should go about doing this.  

https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML
https://developer.mozilla.org/en-US/docs/Learn/Accessibility/CSS_and_JavaScript